### DISCUSSION TITLE
q-table body-cell-name slot with name question

### DISCUSSION THREAD

**Discussion by fanyang-ceg** [quasarframework/quasar#13623] [General - Components / Directives / etc] [answered]
When the column name has space like 'column 001', how to craft the template?  <template v-slot:body-cell-[column 001]="props"> doesn't work

**Comment by MilosPaunovic** [comment]
I'm not sure that's supported.
Do you have any constraints you can't bypass that require a space in a column name?

**Comment by metalsadman** [comment, marked as answer]
https://codepen.io/metalsadman/pen/OJQoNRP it works, but I'd advice against slot names with spaces.

https://vuejs.org/guide/components/slots.html#dynamic-slot-names odd that camelCase wasn't working in the pen tho.

**Comment by fanyang-ceg** [comment]
Thanks @metalsadman, he provided the solution: https://codepen.io/metalsadman/pen/OJQoNRP
